CONTRIBUTING.md

```diff
@@ -219,6 +219,15 @@ python -m pytest test/unit/test_graph.py::GraphTest::test__dependency_list
 ```
 > [Here](https://docs.pytest.org/en/reorganize-docs/new-docs/user/commandlineuseful.html)
 > is a list of useful command-line options for `pytest` to use while developing.
+
+## Adding CHANGELOG Entry
+
+We use [changie](https://changie.dev) to generate `CHANGELOG` entries.  Do not edit the `CHANGELOG.md` directly.  Your modifications will be lost.
+
+Follow the steps to [install `changie`](https://changie.dev/guide/installation/) for your system.
```

Comment:
if we use `make dev` this instruction could change to reflect that.

Reply:
To close the loop from a slack conversation, we can't add changie to `make dev` as it isn't available through pip.  If this becomes an issue we'll explore some alternative ways to make changie available.